src/test/java/org/folio/rest/api/MigrationTestBase.java

```diff
@@ -113,9 +116,24 @@ UpdateResult unsetJsonbProperty(String tableName, UUID id, String propertyName)
       resultSet -> {
         if (resultSet.failed()) {
           result.completeExceptionally(resultSet.cause());
-        } else {
-          result.complete(resultSet.result().getResults());
+          return;
+        }
+        RowIterator<Row> iterator = resultSet.result().iterator();
+        List<JsonArray> list = new LinkedList<>();
+        while (iterator.hasNext()) {
+          Row row = iterator.next();
+          JsonArray ar = new JsonArray();
+          for (int i = 0; i < row.size(); i++) {
+            Object obj = row.getValue(i);
+            if (obj instanceof java.util.UUID) {
+              ar.add(obj.toString());
+            } else {
+              ar.add(obj);
+            }
+          }
```

Comment:
What is this intended to do?

What happens if there is more than one UUID value in a row, are both included in the list?

Reply:
Probably bad things.. I guess the UUID will only bw at a specific column (which we are interested in).

Reply:
The issue is that the response is more typed than Json ... We have to manually map types to Srring.. Such as UUID in this case. The position in the array is "fixed" (columnwise).

Reply:
What do you mean by the position in the array is fixed? The code above seems to interpret every index in the array, unless this has since changed.